src/services/encrypt.ts

```diff
@@ -0,0 +1,25 @@
+import CryptoJS from 'crypto-js';
```

Comment:
I am a little concerned about hashing data using external libraries. It seems to be popular though. Any thoughts on this?
Also, what is the reason for obfuscating user identities? Is it to hide them in our app insights logs? Does Microsoft provide no official method for doing this?